### ISSUE TITLE
Merging nested addon assets into vendor.css

### ISSUE THREAD

**Issue by stavarotti** [ember-cli/ember-cli#3688]
I've looked at the addon docs on site in addition to the addon hooks page in the repo and am unable to find how to achieve merging a nested addon's assets into the app's vendor.css.  Using the `included` hook seems to only work for the addon that is directly referenced in the main app (it does not work for the nested addon).

I've also opened an [SO](http://stackoverflow.com/questions/29311794/nested-addon-css-not-merged-into-vendor-css) question that further describes my setup and intent.  I have not found any docs on how to achieve this hence the post here.

**Comment by stefanpenner** [comment]
I don't believe we can rely on nested addons for this, as they can be duplicated (according to npm's dedupe). This is not compatible with what you want in vendor.css.

How to solve this, and how to think about this is still a topic of further research.

**Comment by stavarotti** [comment]
Thanks.  Is this something that we want documented?  I assume I won't be the only one thinking about nested addons in this fashion given that when running the dummy app during addon development, all appears well.

**Comment by stefanpenner** [comment]
I believe this needs further research before we can even document. We should be confident about are plans here.

This likely should become an rfc we can iterate on 

"Non dup assets of nested addons"

**Comment by stavarotti** [comment]
> I believe this needs further research before we can even document.

Sounds good however, this [section](http://www.ember-cli.com/#addon-nesting) on the change log is a little ambiguous.  It reads (implicitly) as though what I was looking to achieve is possible. 

Lastly, is there a way to achieve my original even if not in vendor.css?

Thanks again Stefan.

**Comment by stavarotti** [comment]
On a similar note, it seems as though the nested component gets merged into the app space.  I assume this is not intended given the this change log statement:

> Prior versions of Ember CLI maintained a flat addon structure, so that all addons (of any depth) would be added to the consuming application.

**Comment by stavarotti** [comment]
Closing since this needs much more research.  In the interim, I am able to get around this by placing the nested addon's assets in the parent addon.

**Comment by ahmed1490** [comment]
@stavarotti Did you copy it manually or did it with some scripts in Blueprint/index.js?
